k8s/overlays/coreso/test/cse-gridcapa-import-runner-itools-coreso-test-configmap.yaml

```diff
@@ -0,0 +1,69 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cse-import-runner-itools-coreso-test-configmap
+  labels:
+    app: gridcapa
+    component: gridcapa-runner
+    region: cse
+
+data:
+  config.yml: |-
+    load-flow:
+      default-impl-name: OpenLoadFlow
+
+    load-flow-default-parameters:
+      voltageInitMode: DC_VALUES
+      balanceType: PROPORTIONAL_TO_GENERATION_P
+      phaseShifterRegulationOn: true
+      countriesToBalance: AL,AT,BA,BE,BG,CH,CZ,DE,ES,FR,GR,HR,HU,IT,ME,MK,NL,PL,PT,RO,RS,SI,SK,UA
+
+    open-loadflow-default-parameters:
+      minPlausibleTargetVoltage: 0.5
+      maxPlausibleTargetVoltage: 1.5
+      plausibleActivePowerLimit: 10000
+      newtonRaphsonConvEpsPerEq: 1.0E-2
+      minRealisticVoltage: 0.4
+      maxRealisticVoltage: 1.6
+
+    rao-objective-function:
+      type: MAX_MIN_MARGIN_IN_AMPERE
+      preventive-stop-criterion: SECURE
+      curative-stop-criterion: SECURE
+
+    rao-range-actions-optimization:
+      max-mip-iterations: 5
+      pst-penalty-cost: 0.01
+      pst-sensitivity-threshold: 0.01
+      pst-model: APPROXIMATED_INTEGERS
+
+    rao-linear-optimization-solver:
+      solver: XPRESS
+      relative-mip-gap : 0.001
+      solver-specific-parameters : THREADS 1 MAXNODE 100000 MAXTIME 300
+
+    rao-multi-threading:
+      contingency-scenarios-in-parallel: 14
```

Comment:
For contingency-scenarios and preventive-leaves, we put only 10 on Coreso test for cse export and swe. I think because the request of cpu was 10. I don't know if we need to increase the cpu of rao pods for that or it's not impacting